recipes-bsp/sgx/files/0021-fix-include-path.patch

```diff
@@ -1,17 +1,12 @@
-Use correct sysroot in include path for cross-compilation
-
-Upstream-Status: Pending
-
 diff --git a/sdk/protected_code_loader/Makefile b/sdk/protected_code_loader/Makefile
-index 5b2bc83..aa20abf 100644
 --- a/sdk/protected_code_loader/Makefile
 +++ b/sdk/protected_code_loader/Makefile
 @@ -84,8 +84,7 @@ PCL_INCLUDE_PATH	:= -I$(COMMON_DIR)/inc \
                         -I$(COMMON_DIR)/inc/internal \
                         -I./crypto \
                         -I./unseal \
--                       -I/usr/include/x86_64-linux-gnu \
 -                       -I/usr/include \
+-                       -I/usr/include/x86_64-linux-gnu \
 +                       -I=/usr/include \
```

Comment:
are we invoking host location here or its on yocto workspace ?

Reply:
From the gcc documentation. If dir begins with ‘=’ or $SYSROOT, then the ‘=’ or $SYSROOT is replaced by the sysroot prefix;. This is from sysroot.

Reply:
it will be good to have $SYSROOT in that case, it for better readability , but it up to you .

Reply:
This is updated from pcl_Makefile.patch for this SGX version.

Reply:
ok